spec/services/recipient_service_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe HandleProjectRecipientService do
+  describe '.call' do
+    
+    let(:recipient) { 
+      PagarMe::Recipient.create(
```

Comment:
Não acho que é uma boa chamar o serviço do pagarme aqui... Se a resposta do Pagar.me for alterada pode quebrar teu teste, sem contar que também vai atrasar ainda mais a suite de testes.

Acho que o que tu pode fazer é mockar essa API e verificar a chamada dos métodos corretamente